### PR TITLE
add a section in troubleshotting docs to check vsphere cloud controll…

### PR DESCRIPTION
**Pull request by abhinavmpandey08**
…er logs

*Issue #, if available:*

*Description of changes:*

*Testing (if applicable):*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/content/en/docs/tasks/troubleshoot/_troubleshooting.md [hunk 1]
```diff
@@ -232,6 +232,12 @@ If this IP is not present on any control plane VM, make sure the `network` has a
 
 {{% content "../../reference/vsphere/domains.md" %}}
 
+If the IPv4 IPs are assigned to the VM and you have the workload kubeconfig under `<cluster-name>/<cluster-name>-eks-a-cluster.kubeconfig`, you can use it to check `vsphere-cloud-controller-manager` logs.
```
Comment (jaxesn) [inline docs/content/en/docs/tasks/troubleshoot/_troubleshooting.md:235]: Can we call out the specific error message that could indicate a misconfig of the VMs having access to the vcenter api?

Comment (chrisnegus) [reply docs/content/en/docs/tasks/troubleshoot/_troubleshooting.md:235]: I agree. Maybe you could show a snippet of output that illustrates what the message might look like.

Comment (abhinavmpandey08) [reply docs/content/en/docs/tasks/troubleshoot/_troubleshooting.md:235]: done

**Review by chrisnegus — COMMENTED** [review]
/lgtm

**Pull request merged**